link/src/test/java/com/stripe/android/link/ui/wallet/PaymentDetailsListItemScreenShotTest.kt

```diff
@@ -0,0 +1,180 @@
+package com.stripe.android.link.ui.wallet
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.core.model.CountryCode
+import com.stripe.android.link.theme.DefaultLinkTheme
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.ConsumerPaymentDetails
+import com.stripe.android.model.CvcCheck
+import com.stripe.android.screenshottesting.FontSize
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.screenshottesting.SystemAppearance
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+@RunWith(Parameterized::class)
+internal class PaymentDetailsListItemScreenShotTest(
+    private val testCase: TestCase
+) {
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        SystemAppearance.entries,
+        FontSize.entries,
+        boxModifier = Modifier
+            .padding(0.dp)
+            .fillMaxWidth(),
+    )
+
+    @Test
+    fun test() {
+        paparazziRule.snapshot {
+            DefaultLinkTheme {
+                PaymentDetailsListItem(
+                    paymentDetails = testCase.state.details,
+                    enabled = testCase.state.enabled,
+                    isSelected = testCase.state.isSelected,
+                    isUpdating = testCase.state.isUpdating,
+                    onClick = {},
+                    onMenuButtonClick = {}
+                )
+            }
+        }
+    }
+
+    companion object {
+        @JvmStatic
+        @Parameterized.Parameters(name = "{0}")
+        @SuppressWarnings("LongMethod")
+        fun data(): List<TestCase> {
```

Comment:
for the screenshot tests where there's only one parameter, it doesn't seem to give us many benefits to make the test parameterized. I think it would make it harder for us to track down a test case that's failing and is also a bit less readable than e.g. using a helper function which would enable us to include just the relevant parameters for that test case